vendor/github.com/openshift/source-to-image/pkg/docker/test/client.go

```diff
@@ -0,0 +1,146 @@
+package test
```

Comment:
probably so, looks like that script is already trying to exclude all test-only files and packages